gemfire-core/src/main/java/com/gemstone/gemfire/cache/client/internal/AbstractOp.java

```diff
@@ -47,7 +47,7 @@
   private final Message msg;
 
   protected AbstractOp(LogWriterI18n lw, int msgType, int msgParts) {
-    this.msg = new Message(msgParts, Version.CURRENT);
+    this.msg = new Message(msgParts, Version.CURRENT_GFE);
```

Comment:
So the assumption is our CURRENT_GFE Version will be compatible with their server version. Assuming they are using a newer or equal version from what we forked out?